docs/jade-getting-started.md

```diff
@@ -52,6 +52,18 @@ relevant to your team.
 Connect your Docker Hub account to your Broad profile by contacting the DevOps
 team.
 
+### 3. Create Terra Accounts
+The Data Repo uses [Sam](https://github.com/broadinstitute/sam) for identity and access management. To register
+as a new user, create an account through Terra. Use a non-Broad email for the non-prod instances:
```

Comment:
At least when we were working w/ just the stewards group, we always added everyone on the team to the stewards group. I'm actually not sure if we would want to do that with the new admins group. But, maybe a helpful instruction here would be to "Request for teammate to add your non-broad email to the relevant groups in SAM for each environment".

Reply:
WHOOPS - didn't see this 5 lines down! Sorry!